File: Logistics.Data/ItemDecommissionVerification.cs
using System;

namespace Logistics.Data
{
    public class ItemDecommissionVerification
    {
        public int Id { get; set; }
        public int ItemDecommissionId { get; set; }
        public int UserId { get; set; }
        public string Remarks { get; set; }
        public DateTime VerificationDate { get; set; }

        public ItemDecommission ItemDecommission { get; set; }
        public User User { get; set; }
    }
}

File: Logistics.Web/ClientApp/src/app/models/item-decommission-verification.ts
import { ItemDecommission } from './item-decommission';
import { User } from './user';

export class ItemDecommissionVerification {
  id: number;
  itemDecommissionId: number;
  userId: number;
  remarks: string;
  verificationDate: Date;
  itemDecommission: ItemDecommission;
  user: User;
}
